x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/add_ml_inference_pipeline_modal.tsx

```diff
@@ -55,7 +55,7 @@ export const AddMLInferencePipelineModal: React.FC<AddMLInferencePipelineModalPr
   }, [indexName]);
 
   return (
-    <EuiModal onClose={onClose}>
+    <EuiModal onClose={onClose} style={{ width: 800 }}>
```

Comment:
question: how does this work with smaller screens?

Reply:
Modal will always take the size of the screen when it is smaller.